common/lib/protocol-definitions/package.json

```diff
@@ -11,11 +11,52 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"sideEffects": false,
+	"type": "module",
 	"exports": {
 		".": {
 			"import": {
-				"types": "./lib/index.d.mts",
-				"default": "./lib/index.mjs"
+				"types": "./lib/index.d.ts",
+				"default": "./lib/index.js"
+			},
+			"require": {
+				"types": "./dist/index.d.ts",
+				"default": "./dist/index.js"
+			}
+		},
+		"./public": {
+			"import": {
+				"types": "./lib/protocol-definitions-public.d.ts",
+				"default": "./lib/index.js"
+			},
+			"require": {
+				"types": "./dist/protocol-definitions-public.d.ts",
+				"default": "./dist/index.js"
+			}
+		},
```

Comment:
I would skip the ./public phase for this and just trim.

Reply:
exactly

Reply:
(That was my ask. `exactly` was a poor reply - I think I read Tyler's "or" as "and".)